modules/apps/commerce/commerce-frontend-taglib/src/main/resources/META-INF/resources/compare_checkbox/page.jsp

```diff
@@ -7,27 +7,36 @@
 
 <%@ include file="/compare_checkbox/init.jsp" %>
 
-<div class="compare-checkbox-root" id="<%= rootId %>"></div>
+<c:choose>
+	<c:when test="<%= commerceChannelId == 0 %>">
+		<div class="alert alert-info mx-auto">
```

Comment:
Are we sure we want to include an error message in a tiny little space the size of a checkbox? The product card may look a little wrecked.